app/controllers/contracts_controller.rb

```diff
@@ -26,6 +26,10 @@ def create
     end
   end
 
+  def index
+    render "index.html.erb"
```

Comment:
and feel free to reorganize the order of methods so that they stay in a conventional pattern. Index's usually go on top